source/jormungandr/jormungandr/parking_space_availability/bss/forseti.py

```diff
@@ -0,0 +1,148 @@
+        else:
+            self.organizations = organizations
+
+    def service_caller(self, method, url):
+        try:
+            response = self.breaker.call(method, url, timeout=self.timeout, verify=False)
+            if not response or response.status_code != 200:
+                logging.getLogger(__name__).error(
+                    'Forseti, Invalid response, status_code: {}'.format(response.status_code)
+                )
+                raise BssProxyError('non 200 response')
+            return response
+        except pybreaker.CircuitBreakerError as e:
+            logging.getLogger(__name__).error('forseti service dead (error: {})'.format(e))
+            raise BssProxyError('circuit breaker open')
+        except requests.Timeout as t:
+            logging.getLogger(__name__).error('forseti service timeout (error: {})'.format(t))
+            raise BssProxyError('timeout')
+        except Exception as e:
+            logging.getLogger(__name__).exception('forseti error : {}'.format(str(e)))
+            raise BssProxyError(str(e))
+
+    @cache.memoize(app.config.get(str('CACHE_CONFIGURATION'), {}).get(str('TIMEOUT_FORSETI'), 30))
+    def _call_webservice(self, arguments):
+        url = "{}?{}".format(self.service_url, arguments)
+        data = self.service_caller(method=requests.get, url=url)
+        return data.json()
+
+    def support_poi(self, poi):
+        return True
+
+    def status(self):
+        # return {'network': self.network, 'operators': self.operators}
+        return {
+            'id': six.text_type(self.network),
+            'url': self.service_url,
+            'class': self.__class__.__name__,
+        }
+
+    def feed_publisher(self):
+        return self._feed_publisher
+
+    def _get_informations(self, poi):
+        longitude = poi.get('coord', {}).get('lon', None)
+        latitude = poi.get('coord', {}).get('lat', None)
+        if latitude is None or latitude is None:
+            return Stands(0, 0, StandsStatus.unavailable)
+
+        params_organizations = ''
+        for param in self.organizations:
+            params_organizations += '&organization[]={}'.format(param)
+
+        # /stations?coord=lon%3Blat&distance=self.distance&organization[]=org1&organization[]=org2 ...
+        arguments = 'coord={}%3B{}&distance={}{}'.format(
+            longitude, latitude, self.distance, params_organizations
+        )
+        data = self._call_webservice(arguments)
+
+        if not data:
+            return Stands(0, 0, StandsStatus.unavailable)
+        obj_stations = data.get('stations', [])
+
+        if not obj_stations:
+            return Stands(0, 0, StandsStatus.unavailable)
+        vehicle_count = 0
+        for v in obj_stations[0].get('vehicles'):
+            vehicle_count = vehicle_count + v.get('count', 0)
+
+        stand = Stands(obj_stations[0].get('docks', {}).get('available', 0), vehicle_count, StandsStatus.open)
```

Comment:
Here, instead of using id, code, ref .. we will match using coordinates as well as distance (with a default value). Distance value will be modified later on via configuration in navitia-configurator.